src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/index.jelly

```diff
@@ -67,27 +67,10 @@ THE SOFTWARE.
                     <f:dropdownListBlock title="${%— Advanced/Deprecated —}"/>
                     <local:listSteps quasiDescriptors="${it.getQuasiDescriptors(true)}"/>
                 </f:dropdownList>
-                <j:set var="id" value="${h.generateId()}"/>
```

Comment:
Loses uniquification of the textarea. OTOH https://github.com/jenkinsci/workflow-cps-plugin/blob/6467f4f1b7842ce7d0c3ed7ad01eb62b5098b724/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/index.jelly#L78 already worked only with the singleton https://github.com/jenkinsci/workflow-cps-plugin/blob/6467f4f1b7842ce7d0c3ed7ad01eb62b5098b724/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/index.jelly#L31 so I guess it does not matter.